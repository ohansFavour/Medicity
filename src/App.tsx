import React from 'react'

// styles
import './App.css'

// components
import Introduction from './containers/introduction/Introduction'
import Health from './containers/health/Health'
import Medicity from './containers/medicity/Medicity'
import Services from './containers/services/Services'
import Footer from './components/footer/footer'

const App: React.FC = (): JSX.Element => {
  return (
    <div className="App">
      <Introduction />
      <Health />
      <Medicity />
      <Services />
      <Footer/>
    </div>
  )
}

export default App
