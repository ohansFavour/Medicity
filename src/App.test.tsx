import React from 'react'
import { shallow } from 'enzyme'

import App from './App'

describe('App Component', () => {
  const component = shallow(<App />)
  test('renders component', () => {
    expect(component).toMatchSnapshot()
  })

  it('renders div with classname App', () => {
    const container = component.find('.App')
    expect(container.length).toBe(1)
  })
})
