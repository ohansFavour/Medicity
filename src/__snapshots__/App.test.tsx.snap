// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders component 1`] = `
<div
  className="App"
>
  Hi
</div>
`;
